supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,66 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Benefits of internal linking
+* Users first: Allows users to navigate intuitively.
+* Allows Google to understand architecture of the site.
+* Establishes a quantifiable pathway to recognize topical expertise.
+* They increase site visitation time.
+* Helps Google crawl your site.
+* Builds relevancy and ranking signals for keywords and topics.
+* Builds citation or helps users add context.
+
+.Technical best practices
+* Use standard hyperlinks:
+`<a href="http://www.same-domain.com/" title="Keyword Text">Keyword Text</a>`
```

Comment:
Does this apply in product documentation? We use link: and xref: in AsciiDoc, not explicitly specifying the `<a></a>` HTML.

Reply:
I'm not sure. I removed the <a></a> here but even then, in openstack at least, we don't use 'href', we use for example:

https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/{osp_curr_ver}/html/overcloud_parameters/policy-parameters["Policy Parameters"]